packages/in_app_purchase/lib/src/in_app_purchase_connection/product.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:in_app_purchase/store_kit_wrappers.dart';
```

Comment:
(optional) I think maybe we should implement this and the other `in_app_purchase_connection/` level changes in a followup patch. It's hard to judge how this should look without both APIs in.